frontend/packages/operator-lifecycle-manager/integration-tests-cypress/tests/operator-hub.cy.ts

```diff
@@ -4,30 +4,25 @@ import { nav } from '../../../integration-tests-cypress/views/nav';
 describe('Interacting with OperatorHub', () => {
   before(() => {
     cy.login();
-    cy.visit('/');
-    nav.sidenav.switcher.changePerspectiveTo('Administrator');
-    nav.sidenav.switcher.shouldHaveText('Administrator');
-    cy.createProject(testName);
+    cy.createProjectWithCLI(testName);
   });
 
   afterEach(() => {
     checkErrors();
   });
 
   after(() => {
-    cy.deleteProject(testName);
+    cy.deleteProjectWithCLI(testName);
     cy.logout();
   });
 
   it('displays OperatorHub tile view with expected available Operators', () => {
     cy.log('navigate to OperatorHub');
     nav.sidenav.clickNavLink(['Operators', 'OperatorHub']);
-    cy.url().should('include', `/operatorhub/ns/${testName}`);
+    cy.url().should('include', '/operatorhub/all-namespaces');
```

Comment:
What changed here?

Reply:
The new method of creating a project with the command line means the newly created project is not selected in the project picker (the existing test depended on project creation method).